python/lsst/ctrl/mpexec/cli/opt/optionGroups.py

```diff
@@ -110,20 +108,9 @@ def __init__(self) -> None:
             ctrlMpExecOpts.skip_existing_option(),
             ctrlMpExecOpts.clobber_outputs_option(),
             ctrlMpExecOpts.save_qgraph_option(),
-            ctrlMpExecOpts.save_single_quanta_option(),
             ctrlMpExecOpts.qgraph_dot_option(),
             ctrlMpExecOpts.qgraph_mermaid_option(),
             ctrlMpExecOpts.summary_option(),
-            ctrlMpExecOpts.save_execution_butler_option(),
-            ctrlMpExecOpts.clobber_execution_butler_option(),
-            ctrlMpExecOpts.target_datastore_root_option(),
```

Comment:
I had forgotten about this option. I think @PaulPrice added it for PFS so it could write files to a different place. I don't know if we have an equivalent for qbb.